content/pl/docs/tutorials/hello-minikube.md

```diff
@@ -0,0 +1,284 @@
+---
+title: Hello Minikube
+content_template: templates/tutorial
+weight: 5
+menu:
+  main:
+    title: "Jak zacząć?"
+    weight: 10
+    post: >
+      <p>Jesteś gotowy ubrudzić ręce? Zbuduj własny klaster kubernetes z działającą na nim aplikacją "Hello World" w Node.js.</p>
+card: 
+  name: tutorials
+  weight: 10
+---
+
+{{% capture overview %}}
+
+Ten samouczek pokaże, jak uruchomić prostą aplikację Hello World w Node.js
+na Kubernetes przy użyciu [Minikube](/docs/setup/learning-environment/minikube) oraz Katacoda.
+Katacoda to darmowe środowisko Kubernetes dostępne bezpośrednio z przeglądarki web.
+
+{{< note >}}
+Możesz też skorzystać z tego samouczka, jeśli już zainstalowałeś [Minikube lokalnie](/docs/tasks/tools/install-minikube/).
+{{< /note >}}
+
+{{% /capture %}}
+
+{{% capture objectives %}}
+
+* Skonfiguruj aplikację *hello world* do uruchomienia w Minikube.
+* Uruchom aplikację.
+* Przejrzyj jej logi.
+
+{{% /capture %}}
+
+{{% capture prerequisites %}}
+
+W tym samouczku wykorzystamy obraz kontenera zbudowany z następujących plików:
+
+{{< codenew language="js" file="minikube/server.js" >}}
+
+{{< codenew language="conf" file="minikube/Dockerfile" >}}
+
+Więcej informacji na temat polecenia `docker build` znajdziesz w [dokumentacji Dockera](https://docs.docker.com/engine/reference/commandline/build/).
+
+{{% /capture %}}
+
+{{% capture lessoncontent %}}
+
+## Stwórz klaster Minikube
+
+1. Kliknij w **Launch Terminal** 
+
+    {{< kat-button >}}
+
+    {{< note >}}Jeśli masz Minikube zainstalowane lokalnie, uruchom `minikube start`.{{< /note >}}
+
+2. Otwórz panel Kubernetes w przeglądarce:
+
+    ```shell
+    minikube dashboard
+    ```
+
+3. Tylko w Katacoda: Na górze okienka z terminalem kliknij na znak plus, a następnie wybierz **Select port to view on Host 1**.
+
+4. Tylko w Katacoda: Wpisz `30000`i kliknij **Display Port**.
+
+## Stwórz Deployment
+
+[*Pod*](/docs/concepts/workloads/pods/pod/) w Kubernetes to grupa jednego lub wielu kontenerów połączonych ze sobą
+z na potrzeby administrowania i dostępu sieci. W tym samouczku Pod zawiera tylko jeden kontener.
+[*Deployment*](/docs/concepts/workloads/controllers/deployment/) w Kubernetes monitoruje stan twojego Poda
+i restartuje należący do niego kontener, jeśli ten z jakichś powodów przestanie działać.
+Użycie Deploymentu to rekomendowana metoda zarządzania tworzeniem i skalowaniem Podów.
+
+1. Użyj polecenia `kubectl create` do stworzenia Deploymentu, który będzie zarządzał Podem. Pod uruchamia kontener
+wykorzystując podany obraz Dockera.
+
+    ```shell
+    kubectl create deployment hello-node --image=gcr.io/hello-minikube-zero-install/hello-node
+    ```
+
+2. Sprawdź stan Deploymentu:
```

Comment:
```suggestion
2. Sprawdź stan Deploymentów:
```

Reply:
This is one deployment only. Why to change to plural?

Reply:
because next line states `get deployments`, plural, and not `get deployment` which is singular.

Reply:
but, you want to check your own, specific deployment. I wouldn't change.